newIDE/app/src/MainFrame/EditorContainers/HomePage/GetStartedSection/UserSurveyQuestion.js

```diff
@@ -0,0 +1,421 @@
+    case 'small':
+      return 1;
+    case 'medium':
+      return 2;
+    case 'large':
+      return 3;
+    case 'xlarge':
+    default:
+      return 4;
+  }
+};
+
+export const TitleAndSubtitle = ({
+  i18n,
+  text,
+  multi,
+  answers,
+  textAlign,
+}: {
+  i18n: I18nType,
+  text: MessageDescriptor,
+  multi: ?boolean,
+  answers: AnswerData[],
+  textAlign: 'center' | 'left',
+}) => (
+  <>
+    <Text size="block-title" align={textAlign}>
+      {i18n._(text)}
+    </Text>
+    {multi ? (
+      <Text style={styles.subTitle} align={textAlign}>
+        {i18n._(t`You can select more than one.`)}
+      </Text>
+    ) : isOnlyOneFreeAnswerPossible(answers) ? (
+      <Text style={styles.subTitle} align={textAlign}>
+        {i18n._(
+          t`The more descriptive you are, the better we can match the content we’ll recommend.`
+        )}
+      </Text>
+    ) : null}
+    <Spacer />
+  </>
+);
+
+const styles = {
+  answerButton: {
+    border: 'solid',
+    borderWidth: 1,
+    borderRadius: 8,
+    width: '100%',
+    height: '100%',
+    display: 'flex',
+  },
+  answerButtonBackground: { width: '100%', height: '100%' },
+  answerCoverImage: {
+    borderTopLeftRadius: 8,
+    borderTopRightRadius: 8,
+    width: '100%',
+  },
+  answerCheckboxAnchor: { position: 'relative' },
+  answerCheckboxContainer: {
+    position: 'absolute',
+    left: 5,
+    top: 'calc(50% - 9px)',
+  },
+  answerTextContainer: { marginLeft: 25, marginRight: 25 },
+  freeAnswerContent: {
+    display: 'flex',
+    height: '100%',
+    flexDirection: 'column',
+  },
+  freeAnswerInputOutline: {
+    border: 'solid',
+    borderWidth: 1,
+    borderRadius: 4,
+    flex: 1,
+    padding: 8,
+  },
+  subTitle: { opacity: 0.6 },
+};
+
+type FreeAnswerProps = {|
+  answerData: AnswerData,
+  onSelect: string => void,
+  selected: boolean,
+  i18n: I18nType,
+  showCheckbox: boolean,
+  onClickSend?: string => void,
+  value: string,
+  onChange: string => void,
+|};
+
+const FreeAnswer = ({
+  answerData,
+  i18n,
+  onSelect,
+  selected,
+  showCheckbox,
+  onClickSend,
+  value,
+  onChange,
+}: FreeAnswerProps) => {
+  const { text, imageSource, id } = answerData;
+  const [errorText, setErrorText] = React.useState<React.Node>(null);
+  const muiTheme = useTheme();
+  const borderColor = (muiTheme.palette.type === 'dark' ? darken : lighten)(
+    muiTheme.palette.text.primary,
+    selected ? 0 : 0.7
+  );
+
+  const clickSend = onClickSend
+    ? () => {
+        setErrorText(null);
+        const cleanedInputValue = value.trim();
+        if (!cleanedInputValue) {
+          setErrorText(<Trans>Please explain your use of GDevelop.</Trans>);
+          return;
+        }
+        onClickSend(cleanedInputValue);
+      }
+    : null;
+
+  return (
+    <ButtonBase
+      style={{
+        ...styles.answerButton,
+        borderColor,
+      }}
+      onClick={e => {
+        if (e.nativeEvent && e.nativeEvent.x === 0 && e.nativeEvent.y === 0) {
```

Comment:
that feels like a hack, but unsure about a better way to handle an input in a button...

Reply:
It is indeed a hack. And I can't remove the `ButtonBase` since the user might click on the card to unselect the answer. So not sure what to do else.

Reply:
ok! this could result in unexpected behaviors on different browsers/devices.
Let's ensure we test that properly, otherwise it's fine 👍